### PR TITLE
Adding the Contributors section

### PR DESCRIPTION
**Pull request by Aggror**
This PR adds a contributor section to the official docs.

Todo
- [x] Add menu items
- [x] Add Docs from Official Stride repo
- [x] Add contributor pages from wiki
- [x] Recheck if those Official contributor docs are valid docs
- [x] Recheck if those  Wiki pages are valid docs
- [x] Check if there are missing pages, and make new Issues for them on the docs.
- [x] Move Stride github docs wiki to contributors section
- [x] Move Stride github Website wiki to contributors section
- [x] Verify menus and links

### CODE REVIEW

**Comment by VaclavElias** [comment]
We can add also the roadmap draft somewhere?

**Comment by Aggror** [comment]
@VaclavElias I think we make mockup of the roadmap after completing both this initial PR and maybe once the docs en website wiki are moved here. What do you think?

**Comment by VaclavElias** [comment]
@Aggror, yes, I agree.

File: en/contributors/engine/architecture/dependency-graph.md
Comment (manio143) [inline en/contributors/engine/architecture/dependency-graph.md:1]: Can you verify this renders correctly in docfx? Images seem broken in PR preview.

File: en/contributors/engine/bug-bounties.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+﻿#bug bounties
```
Comment (manio143) [inline en/contributors/engine/bug-bounties.md:1]: `# Bug bounties`

File: en/contributors/engine/visual-studio-plugin.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+﻿The Stride Visual Studio Plugin adds:
```
Comment (manio143) [inline en/contributors/engine/visual-studio-plugin.md:1]: Missing H1 header `# Visual Studio Plugin`

File: en/contributors/engine/architecture/dependency-graph.md
Comment (Aggror) [reply en/contributors/engine/architecture/dependency-graph.md:1]: I can see them when building locally. But I will specifically have a look here, once we make a test deploy to staging.

**Comment by Aggror** [comment]
@VaclavElias Apart from some texts refering to the Wiki and probably some links, both the Docs and Website Wiki are now residing underneath the Contributor section. I will do some more checking this weekend. Would it be possible to deploy a test to staging?

**Comment by VaclavElias** [comment]
Of course. When do you want me to deploy it? Also, could you push/PR this to a separate branch? I will be able to deploy it from that branch.

In the website repo, you created a branch somehow, but also I can create a branch for you here in Stride docs for this purpose.

image

**Comment by Aggror** [comment]
That was more work than I had anticipated but I think I am almost ready to wrap it up. The final part that is not correct are the source references from build-details.md page. They are not correctly refering to the Stride source.
I have stride docs, site and engine in 3 folders next to each other, but I am missing something.

../../../../../stride/sources/engine/Stride.Engine/Updater/UpdateEngine.cs

**Comment by VaclavElias** [comment]
I will check that path for you today or tomorrow..

**Comment by VaclavElias** [comment]
This can be closed, am I right @Aggror?

**Comment by Aggror** [comment]
This PR has been merged by another PR

**Pull request closed without merging**